src/service/referral-api/acceptance-tests/FamilyHubs.Referral.Api.AcceptanceTests/Tests/Stories/ReferralCountTests.cs

```diff
@@ -0,0 +1,46 @@
+using System.Net;
+using FamilyHubs.Referral.Api.AcceptanceTests.Tests.Steps;
+using TestStack.BDDfy;
+using Xunit;
+
+namespace FamilyHubs.Referral.Api.AcceptanceTests.Tests.Stories;
+
+// Define the story/feature being tested
+[Story(
+    AsA = "user of the referral api",
+    IWant = "to be able to get the count of open referrals",
+    SoThat = "I can know how many referrals there are")]
+public class ReferralCountTests
+{
+    private readonly ReferralCountSteps _steps;
+    private readonly SharedSteps _sharedSteps;
+
+    public ReferralCountTests()
+    {
+        _steps = new ReferralCountSteps();
+        _sharedSteps = new SharedSteps();
+    }
+
+    [Theory]
+    [InlineData("VcsDualRole", "809", HttpStatusCode.OK)] // Happy path as VCS Dual User
+    [InlineData("VcsManager", "809", HttpStatusCode.OK)] // Happy path as VCS Dual User
```

Comment:
nitpick: I think this second comment is meant to be "Happy path as VCS Manager"

Reply:
Good spot, changed